tests/checkpoint/test_checkpoint_result_format.py

```diff
@@ -2372,13 +2372,14 @@ def test_spark_result_format_in_checkpoint_one_multicolumn_map_expectation_compl
     unexpected_index_query: List[int] = evrs[0]["results"][0]["result"].get(
         "unexpected_index_query"
     )
-    assert unexpected_index_query == (
-        "df.filter(F.expr(NOT (((COALESCE(a, 0) + COALESCE(b, 0)) + COALESCE(c, 0)) = "
-        "30)))"
-    )
 
+    assert unexpected_index_query in [
```

Comment:
This test hasn't actually run in a while and I had to change the marker from `filesystem` to `spark`. The test used to check for `30` and produces `30.0`. This should be equivalent so I'm allowing either.